### ISSUE TITLE
V5 | root user password

### ISSUE THREAD

**Issue by osidney** [WPCloudDeploy/wp-cloud-deploy#72]
**Problem**
When creating a LiteSpeed server using version 5.0.0 Beta07 and trying to activate the root user by setting the password, it doesn't work. In this same version try to create a Nginx server and activate the root password from the panel, the same error happened.

**Simulate**
To simulate the error, use version 5.0.0 Beta07, create a server with Nginx or LiteSpeed, access the server, go to the users tab and activate the root password, try to login with the generated password.

**Comment by elindydotcom** [comment]
What provider are you using?  Some providers don't allow logins with passwords.  Other provider will allow it if you explicitly enable password logins but that's a separate step.

**Comment by osidney** [comment]
> What provider are you using? Some providers don't allow logins with passwords. Other provider will allow it if you explicitly enable password logins but that's a separate step.

Hetzner

**Comment by elindydotcom** [comment]
I am unable to replicate this issue.

I just created a new Hetzner server in region fsn1.

I then added a password using the USERS tab for the server.

I was able to login as root using that password.

Are you using a newly created server or an older server?

If you're using an older server, have you perhaps disabled root login?  Maybe try enabling root login using the options on the USER tab just in case it might have been disabled.
